buzzmobile/process/bearing/bearing.py

```diff
@@ -24,13 +24,12 @@ def median(self):
         else:
             return (sortedArray[index] + sortedArray[index + 1]) / 2.0
 
-
 g = {} # globals
 g['last_fix'] = None
-g['med_filter'] = MedianFilter(rospy.get_param('median_filter_size'))
+g['med_filter'] = MedianFilter(rospy.get_param('/buzzmobile/median_filter_size'))
 bearing_pub = rospy.Publisher('bearing', Float64, queue_size=1)
 
-MIN_FIX_DISTANCE = rospy.get_param('min_fix_distance')
+MIN_FIX_DISTANCE = rospy.get_param('/buzzmobile/min_fix_distance')
```

Comment:
You shouldn't need to run everything with params.launch (you just need to have the params loaded, which our other launch files do anyway).

The other alternative is to set defaults with `rospy.get_param('location', DEFAULT)`, which does the same thing more or less.